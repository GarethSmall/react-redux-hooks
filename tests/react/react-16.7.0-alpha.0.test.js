// @flow
import React from 'react';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import { mount } from 'enzyme';
import { MockApp } from '../__fixtures__/react/MockApp';
const { describe, test, expect } = global;

Enzyme.configure({ adapter: new Adapter() });

describe('react-16.7.0-alpha.0.test.js', () => {
  test('should render', () => {
    const component = mount(<MockApp />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
