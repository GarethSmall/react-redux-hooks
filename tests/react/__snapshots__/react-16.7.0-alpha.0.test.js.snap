// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-16.7.0-alpha.0.test.js should render 1`] = `
<MockApp>
  <ReduxProvider>
    <div>
      hi
    </div>
  </ReduxProvider>
</MockApp>
`;
